Caboodle/Compass/Compass.shared.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Threading;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Compass
+    {
+        public static bool IsMonitoring =>
+            MonitorCTS != null && !MonitorCTS.IsCancellationRequested;
+
+        public static void Monitor(SensorSpeed sensorSpeed, Action<CompassData> handler)
```

Comment:
Maybe use `Start` and `Stop`? What does `Monitor` mean? It sounds a bit like watching for errors or something like that.

Reply:
Or maybe `Watch`/`StartWatching` and `StopWatching` - this is naming and I suck at it

Reply:
#resolved